src/main/java/pro/sky/courseworkcouse_2/services/JavaQuestionService.java

```diff
@@ -0,0 +1,76 @@
+package pro.sky.courseworkcouse_2.services;
+
+import org.springframework.stereotype.Service;
+import pro.sky.courseworkcouse_2.interfaces.QuestionService;
+import pro.sky.courseworkcouse_2.models.Question;
+
+import java.util.*;
+
+@Service
+public class JavaQuestionService implements QuestionService {
+
+    private Set<Question> questionStorage;
+
+    private Random random;
```

Comment:
Нужно инициализировать переменную:
```suggestion
    private Random random = new Random();
```